package.json

```diff
@@ -37,6 +37,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.5.5",
+    "@babel/plugin-proposal-class-properties": "^7.5.5",
```

Comment:
This is required to do static props?

Reply:
Yes, `static` is considered experimental since it's still a proposal